src/DynamoCore/Graph/Nodes/NodeModel.cs

```diff
@@ -452,6 +453,82 @@ private string GetCategoryStringFromAttributes()
         }
 
         /// <summary>
+        ///     Dictionary Link property
+        /// </summary>
+        /// <value>
+        ///     If the node has a name and a category, convert them into a link going to the node's help page on
+        ///     Dynamo Dictionary, and return the link.
+        ///     Otherwise, return the Dynamo Dictionary home page.
+        /// </value>
+        [JsonIgnore]
+        public string DictionaryLink
+        {
+            get
+            {
+                string finalLink = Configurations.DynamoDictionary + "#/";
+                if (category == null || category == "")
+                {
+                    return finalLink; // if there is no category, return the link to home page
```

Comment:
I was concerned that `((TextBlock)sender).Tag.ToString()` in `NodeHelpPrompt.xaml.cs` may throw some unexpected errors, but that seems unlikely in this case. Will fix that in my next commit. Thanks!